gtsam/linear/HessianFactor.cpp

```diff
@@ -260,12 +260,12 @@ HessianFactor::HessianFactor(const GaussianFactorGraph& factors,
 void HessianFactor::print(const std::string& s,
     const KeyFormatter& formatter) const {
   cout << s << "\n";
-  cout << " keys: ";
+  cout << " keys: [";
```

Comment:
please undo changes in this file. Touching print in core GTSAM is off limits for now

Reply:
Gotcha. Will save it for later then.